package/MDAnalysis/coordinates/XDR.py

```diff
@@ -43,6 +43,22 @@ def offsets_filename(filename, ending='npz'):
                                                         ending=ending))
 
 
+def read_numpy_offsets(filename):
```

Comment:
Yeah or coordinates/base.  Honestly, I think once this is merged I'm going to pull it all out into a base class for the other formats